public/friendlycode/js/friendlycode.js

```diff
@@ -48,6 +51,7 @@ define(function(require) {
       window: window,
       currentPage: pageToLoad
     });
+    ProjectUI.updateMeta(makeDetails);
```

Comment:
We should be careful using the word `make` still, since we've dropped that idea in the new system.  We only have projects now.